gcpy/raveller_1D.py

```diff
@@ -181,6 +223,29 @@ def has_converged(absdiff, curr_min, n=10):
                               required=True,
                               help='output filename')
 
+    create_site_track = subparsers.add_parser(
+            'create_site_track', help='Generate a track file for ground sites')
+    create_site_track.add_argument('--csv',
+                              type=str,
```

Comment:
Can you align these arguments with the ones from the other add_argument() calls?